pacemaker.spec.in

```diff
@@ -51,7 +52,9 @@
 %bcond_with upstart_job
 
 # Turn off cman support on platforms that normally ship with it
+%if %{cman_allow}
 %bcond_without cman
+%endif
```

Comment:
We don't want to guard this. When someone builds from upstream source, we're adding a --with=cman option, defaulting to without. They can choose CMAN on any platform if they want, we're not going to stop them. The point of the comment is that even if someone builds upstream source on e.g. RHEL 6, CMAN support will be turned off unless they explicitly request it.